depends/packages/rapidcheck.mk

```diff
@@ -1,11 +1,11 @@
 package=rapidcheck
-$(package)_version=3eb9b4ff69f4ff2d9932e8f852c2b2a61d7c20d3
+$(package)_version=d9482c683429fe79122e3dcab14c9655874aeb8e
 $(package)_download_path=https://github.com/emil-e/rapidcheck/archive
 $(package)_file_name=$($(package)_version).tar.gz
 $(package)_sha256_hash=5fbf82755c9a647127e62563be079448ff8b1db9ca80a52a673dd9a88fdb714b
```

Comment:
You'll need to update the `sha256_hash` for this to work.
```bash
shasum -a 256 /Users/michael/Downloads/rapidcheck-d9482c683429fe79122e3dcab14c9655874aeb8e.tar.gz 
b9ee8955b175fd3c0757ebd887bb075541761af08b0c28391b7c6c0685351f6b  /Users/michael/Downloads/rapidcheck-d9482c683429fe79122e3dcab14c9655874aeb8e.tar.gz
```

Reply:
Thank you. Verified the hash on my end and updated.